app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    [Person(person["name"], person["age"]) for person in people]
+
+    for person_data in people:
+        person = Person.people[person_data["name"]]
+        wife_name = person_data.get("wife")
+        husband_name = person_data.get("husband")
+
+        if wife_name and wife_name in Person.people:
+            person.wife = Person.people[wife_name]
```

Comment:
The code assigns `person.wife` without checking if the `wife` attribute exists on the `Person` instance. Consider initializing `wife` and `husband` attributes in the `Person` class constructor to avoid potential attribute errors.